libraries/AP_Mount/LogStructure.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <AP_Logger/LogStructure.h>
+
+#define LOG_IDS_FROM_MOUNT \
+    LOG_MOUNT_MSG
+
+// @LoggerMessage: MNT
+// @Description: Mount's actual and Target/Desired RPY information
+// @Field: TimeUS: Time since system startup
+// @Field: I: Instance number
+// @Field: DesRoll: mount's desired roll
+// @Field: Roll: mount's actual roll
+// @Field: DesPitch: mount's desired pitch
+// @Field: Pitch: mount's actual pitch
+// @Field: DesYawB: mount's desired yaw in body frame
+// @Field: DesYawE: mount's desired yaw in earth frame
+// @Field: YawB: mount's actual yaw in body frame
```

Comment:
I think we should change the order of these Yaw fields to be "DesYawB", "YawB", "DesYawE", "YawE".